central/sensor/service/pipeline/complianceoperatorprofiles/pipeline_test.go

```diff
@@ -145,33 +94,9 @@ func (s *PipelineTestSuite) TestRunV1Delete() {
 func (s *PipelineTestSuite) TestRunReconcileNoOp() {
 	ctx := context.Background()
 
-	pipeline := NewPipeline(s.v1ProfileDS, s.manager, s.v2ProfileDS)
-
-	s.v1ProfileDS.EXPECT().Walk(ctx, gomock.Any()).Return(nil).Times(1)
-	s.v2ProfileDS.EXPECT().GetProfileEdgesByCluster(ctx, fixtureconsts.Cluster1).Return(nil, nil).Times(1)
-
-	err := pipeline.Reconcile(ctx, fixtureconsts.Cluster1, reconciliation.NewStoreMap())
-	s.NoError(err)
-}
-
-func (s *PipelineTestSuite) TestRunReconcile() {
```

Comment:
This test was mostly focused on V2 and didn't exist prior to using V2.  Removing it from here but keeping it in the V2 pipeline test.